src/utils/getFileTypeFromPath.ts

```diff
@@ -0,0 +1,20 @@
+import { FileType } from "../tree/types";
+
+export const getFileTypeFromPath = (path: string): FileType => {
+  const count = (path.match(/\./g) || [])
+    .length; /* find the amount of periods assuming the path is relative and starts with "./" */
+
+  if (count == 3) {
```

Comment:
no magic numbers....
const CONFIG_NUMBER_OF_DOTS = 3

Reply:
Changed `count` to`numberOfDots`